src/CommonGLPI.php

```diff
@@ -696,18 +700,59 @@ public static function displayStandardTab(CommonGLPI $item, $tab, $withtemplate
         return false;
     }
 
+    /**
+     * @param class-string<CommonGLPI>|null $form_itemtype
+     * @return string
+     */
+    private static function getTabIconClass(?string $form_itemtype = null): string
+    {
+        $default_icon = CommonDBTM::getIcon();
+        $icon = $default_icon;
+        $tab_itemtype = static::class;
+        $itemtype = $tab_itemtype;
+
+        if ($icon === $default_icon && is_subclass_of($tab_itemtype, CommonDBRelation::class)) {
+            // Get opposite itemtype than this
+            if ($tab_itemtype::$itemtype_1 === $form_itemtype || ($tab_itemtype::$itemtype_1 === 'itemtype' && $tab_itemtype::$itemtype_2 !== null)) {
+                $itemtype = $tab_itemtype::$itemtype_2;
+            } else if ($tab_itemtype::$itemtype_2 === $form_itemtype || ($tab_itemtype::$itemtype_2 === 'itemtype' && $tab_itemtype::$itemtype_1 !== null)) {
+                $itemtype = $tab_itemtype::$itemtype_1;
+            }
```

Comment:
This is not the first time we have to do this kind of computation. Could you move this in a  `CommonDBRelation::getOppositeItemtype(string $itemtype): string` ?